src/vuln_analysis/eval/evaluators/accuracy.py

```diff
@@ -0,0 +1,221 @@
+                pbar.update(1)
+                return result
+
+        try:
+            tqdm_position = TqdmPositionRegistry.claim()
+            pbar = tqdm(total=len(eval_input.eval_input_items),
+                        desc=f"Evaluating {self.field.title()} Accuracy",
+                        position=tqdm_position)
+            results = await asyncio.gather(*[wrapped_process(item) for item in eval_input.eval_input_items])
+        finally:
+            pbar.close()
+            TqdmPositionRegistry.release(tqdm_position)
+
+        all_averages, all_reasonings, total_items, total_correct = zip(*results) if results else ([], [], [], [])
+
+        # Choose output based on config: return box & whiskers values if multiple_reps=True, else return a single avg
+        if self.multiple_reps:
+            scores_sequential = list(all_averages)
+            if scores_sequential:
+                scores_sequential.sort()
+                # Calculating average by averaging each EvalOutputItem's score since for multiple reps, each EvalOutputItem has the same number of CVEs
+                mean_value = round(sum(scores_sequential) / len(scores_sequential), 2)
+                min_value = round(scores_sequential[0], 2)
+                max_value = round(scores_sequential[-1], 2)
+                try:
+                    q1, med, q3 = stats.quantiles(scores_sequential, n=4, method="inclusive")
+                    q1 = round(q1, 2)
+                    med = round(med, 2)
+                    q3 = round(q3, 2)
+                except Exception:  # Fallback for very small samples that may fail to compute quantiles
+                    med = round(stats.median(scores_sequential), 2)
+                    q1 = med
+                    q3 = med
+                box_and_whiskers_values = [mean_value, min_value, q1, med, q3, max_value]
+            else:
+                box_and_whiskers_values = [0.0, 0.0, 0.0, 0.0, 0.0, 0.0]
+            final_output = box_and_whiskers_values
+        else:  # Single rep
+            # Calculating weighted average (each EvalOutputItem's score is weighted by its number of CVEs)
+            total_correct_sum = sum(total_correct)
+            total_items_sum = sum(total_items)
+            final_output = round(total_correct_sum / total_items_sum, 2) if total_items_sum > 0 else 0.0
+
+        eval_output_items = [
+            EvalOutputItem(id=item.id, score=score, reasoning=reasoning)
+            for item, score, reasoning in zip(eval_input.eval_input_items, all_averages, all_reasonings)
+        ]
+
+        return EvalOutput(average_score=final_output, eval_output_items=eval_output_items)
+
+
+class AccuracyEvaluatorConfig(EvaluatorBaseConfig, name="accuracy"):
+    """Configuration for unified accuracy evaluator"""
+    field: str = "status"  # "status" or "label"
+    multiple_reps: bool = False
+    conflict_policy: str = "keep_first"  # "strict", "keep_first", or "lenient"
```

Comment:
I recommend using `typing.Literal` instead of `str` to constrain the possible values.
It's also a best practice to define configs using pydantic Fields to ensure documentation is populated for the `nat info components` command.

(You'll need to add a couple of imports)
```python
import typing
from pydantic import Field
```

```suggestion
    field: typing.Literal["status", "label"] = Field(default="status", description="The field to evaluate accuracy on.")
    multiple_reps: bool = Field(default=False, description="Whether to evaluate multiple reps of the same input.")
    conflict_policy: typing.Literal["strict", "keep_first", "lenient"] = Field(
        default="keep_first", description="The policy to use for resolving conflicting ground truth values.")
```